handlers/iface/iface.go

```diff
@@ -0,0 +1,14 @@
+package iface
```

Comment:
This package needs a better name, any ideas?

Reply:
Would you be against having this one in the handlers package as it will be used by all the handlers in packages further down? then would just be `handlers.Router` in the other places, that would feel more natuarl to me

Agree that `iface` is far from ideal but will leave up to you - can always change it again

Reply:
I don't think we can do this because of circular dependencies